internal/source/dispatcher.go

```diff
@@ -185,13 +186,17 @@ func (d *Dispatcher) dispatchMsg(ctx context.Context, event source.Event, dispat
 				return
 			}
 
-			reportErr := d.reportSuccess(ctx, n, pluginName, event, dispatch.sourceName)
+			reportErr := d.reportSuccess(n, pluginName, event)
 			if reportErr != nil {
 				d.log.Error(err)
 			}
 		}(n)
 	}
 
+	if err := d.reportAuditEvent(ctx, pluginName, event.RawObject, dispatch.sourceName, dispatch.sourceDisplayName); err != nil {
+		d.log.Errorf("while reporting audit event for source %q: %s", dispatch.sourceName, err.Error())
```

Comment:
what about `d.log.Errorf("while reporting audit event for source %q: %w", dispatch.sourceName, err)`? To not lose error details.

Reply:
To me the `%s` pattern in logging is more readable, but sure, I can change that 👍

Reply:
This is logging, so actually we'll still print just error message. When using `%w`, we still have just an error message, but wrapped in a bit odd format: https://go.dev/play/p/nDVTk9L-VPl 


So `%s` for logging is more readable, and it is used in all over the places 🤔  To get all the stacktrace etc. we'd need to use `%v`, but that would make our logs hard to read 😞